docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md

```diff
@@ -66,17 +66,17 @@ This page shows supported and optimal configurations for each plugin.
 Terminology
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-===================  =============================================
- Acronym/Term         Description                                 
-===================  =============================================
- FP32 format          Single-precision floating-point format       
- BF16 format          Brain floating-point format                  
- FP16 format          Half-precision floating-point format         
- I16 format           2-byte signed integer format                 
- I8 format            1-byte signed integer format                 
- U16 format           2-byte unsigned integer format               
- U8 format            1-byte unsigned integer format               
-===================  =============================================
++------------------+----------------------------------------------+
+| Acronym/Term     |   Description                                |
++==================+==============================================+
+| FP32 format      |   Single-precision floating-point format     |
```

Comment:
We support double precision floating point (FP64) and integer precision too. @ilya-lavrenov where should these be added in this document (input/output/model precision)? For integer, should we list both I64 and I64? Do we miss other formats?

@msmykx-intel these tables are tricky to review! Are there rendered previews for these pages?